proto/store/policy.proto

```diff
@@ -29,3 +30,29 @@ message Binding {
   // If the condition evaluates to false, then this binding does not apply to the current request. However, a different role binding might grant the same role to one or more of the principals in this binding.
   google.type.Expr condition = 3;
 }
+
+// DataMaskingExceptionPolicy is the whitelist of users who can access sensitive data.
+message DataMaskingExceptionPolicy {
+  message DataMaskingException {
+    enum Action {
+      ACTION_UNSPECIFIED = 0;
+      QUERY = 1;
+      EXPORT = 2;
+    }
+    // action is the action that the user can access sensitive data.
+    Action action = 1;
+
+    // Level is the masking level that the user can access sensitive data.
+    ObfusecateLevel level = 2;
+
+    // Members is the list of principals who bind to this exception policy instance.
+    //
+    // * `user:{emailid}`: An email address that represents a specific Bytebase account. For example, `alice@example.com`.
+    repeated string members = 3;
+
+    // The condition that is associated with this exception policy instance.
+    google.type.Expr condition = 4;
+  }
```

Comment:
Do we need the expiration time?

Reply:
The expiration time will be part of condition (CEL string), very close to data access control.